site/content/docs/5.3/utilities/overflow.md

```diff
@@ -90,9 +90,9 @@ Adjust the `overflow-y` property to affect the overflow of content vertically.
 
 Using Sass variables, you may customize the overflow utilities by changing the `$overflows` variable in `_variables.scss`.
 
-## Sass
+## CSS
 
-### Utilities API
+### Sass utilities API
 
 Overflow utilities are declared in our utilities API in `scss/_utilities.scss`. [Learn how to use the utilities API.]({{< docsref "/utilities/api#using-the-api" >}})
```

Comment:
Realizing through this code review that, separately or here, we could perhaps turn this section (heading and paragraph) into an include with an argument lol. Maybe, maybe not.

Reply:
The idea's great. We would need a shortcode or partial or anything else with two params:
- First part of the sentence (e.g. "Background" or "Object fit")
- The param (the key) used for `scss-docs` (e.g. "utils-bg-color")

The issue is I don't know how to build it with Hugo. I tried few things but haven't managed to succeed by calling `scss-docs` and `docsref` within another shortcode or form a partial.

Would like to call it like this:

```
{{< sass-utilities-api-section "Background" "utils-bg-color" >}}
```

And it would contain something like:

```
### Sass utilities API

{{ firstParam }} utilities are declared in our utilities API in `scss/_utilities.scss`. [Learn how to use the utilities API.]({{< docsref "/utilities/api#using-the-api" >}})

{{< scss-docs name="{{ secondParam }}" file="scss/_utilities.scss" >}}
```

Maybe @XhmikosR you'd know how to do it

Reply:
@mdo Maybe we can do it with a step-by-step approach: merge this PR, and create an issue to track this possible enhancement?

Reply:
FYI tracked in https://github.com/twbs/bootstrap/issues/38434